### DISCUSSION TITLE
Crash reporter errors out

### DISCUSSION THREAD

**Discussion by bilderbuchi** [OpenModelica/OpenModelica#10309] [Q&A]
A colleague experienced a crash of OMEdit, and wanted to submit a report. Unfortunately, the crash report reporter failed! I only have a screenshot of the error message:
![image](https://user-images.githubusercontent.com/327442/222667454-0931319e-30ec-4805-b3b3-5a9e11d6994e.png)
Is that a known problem?

Also, we are currently experiencing sudden crashes when translating a model (too big to be an MWE for a bug report). 
On my installation, I never seem to get the prompt to submit a crash report, but he does -- is there a setting -- that I might have inadvertantly set at some point in the past -- that disables the crash report dialog?

**Comment by adeas31** [comment]
There is no setting for crash report. Maybe you are behind some firewall or something. The error says that it failed to transfer.

**Reply by bilderbuchi** [reply]
Yeah, firewall could be a reason for the failure to transfer. Although it must have reached the server for it to reply - does the transfer use a different port?

I was puzzled that I didn't get the dialog at all -- maybe the particularities of the crash experienced. Thanks for confirming that there is no setting for that.

**Reply by adeas31** [reply]
Not getting a crash dialog depends on several things, it might be that we are not checking for some signal that is causing the crash.

Is it possible for you to share the model that is crashing and I will debug more.

**Reply by bilderbuchi** [reply]
The model's not public, but there's a backtrace at https://github.com/OpenModelica/OpenModelica/discussions/10313#discussioncomment-5189154. The problem was an access violation (exit code -1073741819)
